user_sync/config.py

```diff
@@ -445,7 +511,7 @@ def check_unused_config_keys(self):
             if (len(unused_keys) > 0):
                 raise user_sync.error.AssertionException("Unused source filters for: %s" % list(unused_keys))
         
-    
+# base class for configuration tree
```

Comment:
not really a helpful comment, but if you are going to have a comment on a class, you should make it the string documentation (first entry in the class).